katas/content/teleportation/index.md

```diff
@@ -0,0 +1,33 @@
+# Teleportation
+
+@[section]({
+    "id": "teleportation__overview",
+    "title": "Overview"
+})
+
+Teleportation quantum kata is a series of exercises designed to get you familiar with programming in Q#. It covers the quantum teleportation protocol which allows you to communicate a quantum state using only classical communication and previously shared quantum entanglement.
+
+- Teleportation is described in <a href="https://en.wikipedia.org/wiki/Quantum_teleportation">this Wikipedia article</a>.
+- An interactive demonstration can be found <a href="http://demonstrations.wolfram.com/QuantumTeleportation/">on the Wolfram Demonstrations Project</a>.
+Each task is wrapped in one operation preceded by the description of the task. Your goal is to fill in the blank (marked with // ... comment) with some Q# code that solves the task. To verify your answer, run the cell
```

Comment:
This is notebooks instruction (we don't have cells on the website), so we don't need it here - at this point in the katas the learner will have figured out how to use them
```suggestion
```

Reply:
Removed